examples/ssd1306_128x64_i2c/ssd1306_128x64_i2c.ino

```diff
@@ -86,31 +86,33 @@ void setup() {
   // drawing operations and then update the screen all at once by calling
   // display.display(). These examples demonstrate both approaches...
 
-  testdrawline();      // Draw many lines
+  void testdrawline();      // Draw many lines
```

Comment:
It looks like you have changed these from function calls into function declarations.
So now they don't actually do anything.

Reply:
Wow, you are right. No idea why it worked on my system. I didnt even notice all those calls were in setup(). Let me try and figure out how to fix, if not I can cancel this PR and do it properly later.